databricks_cli/pipelines/cli.py

```diff
@@ -80,29 +80,36 @@ def deploy_cli(api_client, spec_arg, spec, allow_duplicate_names, no_update_spec
     OR
```

Comment:
Can you fix the comment above to reflect the new behavior?

Reply:
Done.